Dockerfile

```diff
@@ -91,7 +92,7 @@ RUN gem install \
 RUN rm -rf /usr/local/bundle/cache/* \
  && find /usr/local/bundle/ -name "*.o" | xargs rm
 
-FROM fluent/fluentd:v1.12.2-debian-1.0
+FROM fluent/fluentd:v1.12.2-${FLUENTD_FLAVOR:-debian}-1.0
```

Comment:
We should move default to the `ARG` [link](https://docs.docker.com/engine/reference/builder/#arg)

Reply:
```
ARG FLUENTD_FLAVOR=debian
```

Reply:
```
ARG FLUENTD_FLAVOR=debian
```

Reply:
I've changed the `FLUENTD_FLAVOR` to `FLUENTD_ARCH` build arg, which is empty by default (for amd64 architecture).